src/main/java/uk/gov/hmcts/reform/roleassignmentbatch/writer/EntityWrapperWriter.java

```diff
@@ -7,21 +7,21 @@
 import org.springframework.batch.item.database.JdbcBatchItemWriter;
 import org.springframework.beans.factory.annotation.Autowired;
 import uk.gov.hmcts.reform.roleassignmentbatch.entities.EntityWrapper;
-import uk.gov.hmcts.reform.roleassignmentbatch.entities.Newtable;
+import uk.gov.hmcts.reform.roleassignmentbatch.entities.HistoryEntity;
 import uk.gov.hmcts.reform.roleassignmentbatch.entities.RequestEntity;
 
 public class EntityWrapperWriter implements ItemWriter<EntityWrapper> {
 
     @Autowired
-    private JdbcBatchItemWriter<Newtable> newtableWriter;
+    private JdbcBatchItemWriter<HistoryEntity> historyWriter;
     @Autowired
     private JdbcBatchItemWriter<RequestEntity> requestEntityWriter;
 
     @Override
     public void write(List<? extends EntityWrapper> items) throws Exception {
-        for (EntityWrapper item: items) {
-            newtableWriter.write(Collections.singletonList(item.getNewtable()));
+        for (EntityWrapper item : items) {
             requestEntityWriter.write(Collections.singletonList(item.getRequestEntity()));
+            historyWriter.write(Collections.singletonList(item.getHistoryEntity()));
         }
```

Comment:
are we not writing the live table record?

Reply:
this live table code is already available on base branch